### PR TITLE
Ability to pass additional comments to ply header.

### PR DESCRIPTION
**Pull request by feuerste**

### CODE REVIEW

File: cartographer/io/ply_writing_points_processor.cc [hunk 1]
```diff
@@ -86,18 +90,22 @@ PlyWritingPointsProcessor::FromDictionary(
     common::LuaParameterDictionary* const dictionary,
     PointsProcessor* const next) {
   return absl::make_unique<PlyWritingPointsProcessor>(
-      file_writer_factory(dictionary->GetString("filename")), next);
+      file_writer_factory(dictionary->GetString("filename")),
+      std::vector<std::string>(), next);
```
Comment (CodeArno) [inline cartographer/io/ply_writing_points_processor.cc]: Nit: Instead of `std::vector<std::string>()`, you can just write `{}`.

**Comment by feuerste** [comment]
@wally-the-cartographer merge

**Comment by feuerste** [comment]
Thanks for the review @CodeArno

**Comment by wally-the-cartographer** [comment]
Merge requested by authorized user feuerste. Merge queue now has a length of 1.

**Comment by wally-the-cartographer** [comment]
Error: PUT https://api.github.com/repos/googlecartographer/cartographer/pulls/1549/merge: 405 At least 1 approving review is required by reviewers with write access. []

Log:
~~~
=> handle work item: PR 1549 on googlecartographer/cartographer.
 => Switching to branch master.
 $ git reset --hard HEAD [data/cartographer]
HEAD is now at a6050b58 Update README.rst (#1542)
 $ git clean -fd [data/cartographer]
 $ git checkout master [data/cartographer]
Already on 'master'
Your branch is up to date with 'origin/master'.
 $ git pull [data/cartographer]
Already up to date.
 => Checking out googlecartographer/mfeuerstein/ply_comments as local branch pr_1549.
 $ git remote add googlecartographer git@github.com:googlecartographer/cartographer.git [data/cartographer]
fatal: remote googlecartographer already exists.
 $ git fetch googlecartographer [data/cartographer]
 $ git branch --track pr_1549 googlecartographer/mfeuerstein/ply_comments [data/cartographer]
Branch 'pr_1549' set up to track remote branch 'mfeuerstein/ply_comments' from 'googlecartographer'.
 => Switching to branch pr_1549.
 $ git reset --hard HEAD [data/cartographer]
HEAD is now at a6050b58 Update README.rst (#1542)
 $ git clean -fd [data/cartographer]
 $ git checkout pr_1549 [data/cartographer]
Switched to branch 'pr_1549'
Your branch is up to date with 'googlecartographer/mfeuerstein/ply_comments'.
 $ git pull --force [data/cartographer]
Already up to date.
 => merge master.
 $ git merge master [data/cartographer]
Already up to date.
 => Running clang-format on all *.cc and *.h files
 $ git commit -am Ran clang-format. [data/cartographer]
On branch pr_1549
Your branch is up to date with 'googlecartographer/mfeuerstein/ply_comments'.

nothing to commit, working tree clean
 $ git push --force googlecartographer HEAD:mfeuerstein/ply_comments [data/cartographer]
Everything up-to-date
 => Abandoning branch pr_1549.
 => Switching to branch master.
 $ git reset --hard HEAD [data/cartographer]
HEAD is now at 3a8f7d2e Add newline to comment.
 $ git clean -fd [data/cartographer]
 $ git checkout master [data/cartographer]
Switched to branch 'master'
Your branch is up to date with 'origin/master'.
 $ git branch -D pr_1549 [data/cartographer]
Deleted branch pr_1549 (was 3a8f7d2e).

~~~

**Comment by CodeArno** [comment]
@wohe Can you please approve?

**Comment by feuerste** [comment]
@wally-the-cartographer merge

**Comment by wally-the-cartographer** [comment]
Merge requested by authorized user feuerste. Merge queue now has a length of 1.

**Pull request merged**